components/home/ChartBox/Chart.tsx

```diff
@@ -123,18 +124,66 @@ const Chart = (props: ChartProps) => {
         snapTooltipToDatumX
         snapTooltipToDatumY
         showDatumGlyph
-        applyPositionStyle={true}
+        showVerticalCrosshair
+        verticalCrosshairStyle={{ stroke: '#B8B8B8', strokeDasharray: 3 }}
+        showSeriesGlyphs
+        applyPositionStyle
         style={{
           background: 'transparent',
         }}
         unstyled={false}
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        renderTooltip={({ tooltipData }: any) => (
-          <div className="bg-tooltipBackground px-[8px] py-[4px] rounded-[8px] text-baseForeground text-12">
-            {label && <p className="text-10 text-neutral-200">{label}</p>}
-            {AmountConverter(tooltipData.nearestDatum.datum.count)}
-          </div>
-        )}
+        renderTooltip={({ tooltipData }: any) => {
+          console.log({ tooltipData });
+          const currentPeriodKey = `Current ${
+            currentFilter ? currentFilter.name : ''
+          }`;
+          const prevPeriodKey = `Prev ${
+            currentFilter ? currentFilter.name : ''
+          }`;
+          return (
+            <div className="bg-tooltipBackground px-[8px] py-[4px] rounded-[8px] text-neutral-200 text-xs">
+              <span className="text-[10px]">
```

Comment:
please use font size of tailwind instead of text-[10px] and text-xs

Reply:
fixed